apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -188,3 +205,47 @@ function ProgressBar(partial: number | null, total: number | null) {
     </div>
   `;
 }
+
+function GraderAssignmentModal({
+  csrfToken,
+  courseStaff,
+}: {
+  csrfToken: string;
+  courseStaff: User[];
+}) {
+  return Modal({
+    id: 'grader-assignment-modal',
+    title: 'Assign instances to graders',
+    body: html`
+      <p>Assign instances to the following graders:</p>
+      ${courseStaff.map(
+        (staff) => html`
+          <div class="form-check">
+            <input
+              type="checkbox"
+              id="grader-assignment-${staff.user_id}"
+              name="assigned_grader"
+              value="${staff.user_id}"
+              class="form-check-input"
+            />
+            <label class="form-check-label" for="grader-assignment-${staff.user_id}">
+              ${staff.name ? `${staff.name} (${staff.uid})` : staff.uid}
+            </label>
+          </div>
+        `,
+      )}
```

Comment:
Nit, consider adding an empty state when there are no grader users:

<img width="831" alt="Screenshot 2024-08-15 at 10 40 09" src="https://github.com/user-attachments/assets/9991b770-a877-40bc-b4a8-56d6588e5170">

Reply:
Done in 72dba21b71bb681b019777070e4c735e9b8711be.